examples/applications/plot_out_of_core_classification.py

```diff
@@ -398,7 +404,10 @@ def autolabel(rectangles):
 cls_names.append('Read/Parse\n+Feat.Extr.')
 cls_runtime.append(vectorizing_time)
 cls_names.append('Hashing\n+Vect.')
-bar_colors = rcParams['axes.color_cycle'][:len(cls_names)]
```

Comment:
You can actually remove this line completely since bar_colors is defined earlier.